Volume/Views/BrowserView.swift

```diff
@@ -12,10 +12,13 @@ import WebKit
 
 struct BrowserView: View {
     @EnvironmentObject private var userData: UserData
+    @State var shoutoutsButtonEnabled: Bool = true
```

Comment:
I don't think `shoutoutsButtonEnabled` is being used right now. While you're updating it in the `.onAppear` and in `incrementShoutouts()`, it isn't used for the actual Button. So maybe replace this w/ the `isShoutoutsButtonEnabled` and make it a computed var as Daniel said. Then you can get rid of the `.onAppear`, line 22 and line 66.

Reply:
yeah thats what I ended up doing.  Thanks!